operations/system_admin_toolkit/usage/Change_CFS_Version.md

```diff
@@ -0,0 +1,68 @@
+# Change the CFS Version
+
+By default, SAT uses Configuration Framework Service (CFS) version three (`v3`).
+Currently, only CFS version `"v2"` can be used with `sat bootsys` and `sat bootprep`.
+Select the CFS version to use for individual commands with the `--cfs-version`
+option. For more information on this option, refer to the man page for a specific
+command.
```

Comment:
If I understand correctly, currently the version of the CFS API used by `sat bootsys` and `sat bootprep` are hard-coded to v2, so those commands are not affected by the CLI or config-file option described below. I think we could make that a little more clear. I hope we'll be able to fix this in the very near future in CRAYSAT-1896.

Here's my suggestion to make this more clear until we address CRAYSAT-1896.

```suggestion
SAT supports specifying the version of the Configuration Framework Service (CFS)
API used by the `sat status` command. By default the `sat status` command uses
version three (V3) of the CFS API. Currently, the `sat bootsys` and `sat
bootprep` commands can only use CFS V2, but support will be added for CFS V3 in
the future.
```

Reply:
Addressed